tests/admin_inlines/tests.py

```diff
@@ -1372,7 +1372,7 @@ def test_inlines_verbose_name(self):
             'admin:admin_inlines_courseproxy2_add',
         ]
         css_selector = '.dynamic-class_set#class_set-%s h2'
-
+        self.selenium.set_window_size(1024, 768)
```

Comment:
I think it would be batter to hide a sidebar, e.g.
```python
        # Hide sidebar.
        toggle_button = self.selenium.find_element_by_css_selector('#toggle-nav-sidebar')
        toggle_button.click()
```